### PR TITLE
Steps to use metalb and beginning of usage doc

### PR DESCRIPTION
**Pull request by cooktheryan**
Initial commit of usage doc and metal lb instructions

### CODE REVIEW

File: content/usage/README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    - name: default
+      protocol: layer2
+      addresses:
+      - 192.168.1.240-192.168.1.250
+```
+
+Load the configMap into the cluster.
+```
+kubectl create -f /tmp/configmap.yaml
+```
```
Comment (mangelajo) [inline content/usage/README.md:40]:
```suggestion
Create the metallb configmap:
```
kubectl create -f - <<EOF
apiVersion: v1
kind: ConfigMap
metadata:
  namespace: metallb-system
  name: config
data:
  config: |
    address-pools:
    - name: default
      protocol: layer2
      addresses:
      - 192.168.1.240-192.168.1.250
EOF
```
```

Comment (cooktheryan) [reply content/usage/README.md:40]: I like it! Will fix!

**Review by mangelajo — APPROVED** [review]

File: content/usage/README.md (hunk 1 above)
Comment (mangelajo) [reply content/usage/README.md:40]: ack, Also for the other, vim we can do the same thing.

**Comment by mangelajo** [comment]
I'm testing locally and it works, they even seem to have multiarch images!!! :rocket:

**Comment by cooktheryan** [comment]
Woot that's even better that they are multi-arch!!!

**Pull request merged**